R/vdiagram-render.R

```diff
@@ -54,6 +54,129 @@ renderVDiagram <- function(data){
 }
 
 createVdiagram <- function(data) {
-  data <- parseVDiagramData(data)
-  vplot(data)
+  styles <- getVDiagramStyle()
+  
+  vdiagramData <- parseVDiagramData(data)
+  
+  vplot <- gsplot() %>%
+    points(NA,NA, ylab=styles$labels$y, xlab=styles$labels$x) %>%
+    callouts(y=c(vdiagramData$minStage, vdiagramData$maxStage), styles$callouts) %>%
+    grid(styles$grid) %>%
+    axis(styles$axis);
+  
+  vplot <- addMeasurementsAndError(vplot, vdiagramData, styles)
+  vplot <- addRatingShifts(vplot, vdiagramData, styles)
+  
+  if (any(!is.na(vdiagramData$obsShift)) && any(!vdiagramData$histFlag)){
+    vplot <- callouts(vplot,x = vdiagramData$obsShift[!vdiagramData$histFlag], y = vdiagramData$obsGage[!vdiagramData$histFlag], 
+                      labels=vdiagramData$obsCallOut[!vdiagramData$histFlag], cex=0.6)
+  }
+  
+  par(mar=c(7, 3, 4, 2))
```

Comment:
Not sure if y'all have tried this, but the way we hope to bake in the high level par calls in gsplot is through the initial call. So up on line 61, it would be 
```R
vplot <- gsplot(mar=c(7, 3, 4, 2)) %>%
  # bla bla
```